backend/src/main/java/org/eclipse/tractusx/puris/backend/stock/logic/adapter/ItemStockSammMapper.java

```diff
@@ -117,21 +115,20 @@ private record PositionsMappingHelper(Date date, String customerOrderId, String
 
 
     public List<ReportedProductItemStock> itemStockSammToReportedProductItemStock(ItemStockSamm samm, Partner partner) {
-        String matNbrCustomer = samm.getMaterialNumberCustomer();
-        String matNbrSupplier = samm.getMaterialNumberSupplier(); // should be ownMaterialNumber
-        String matNbrCatenaX = samm.getMaterialGlobalAssetId();
+        String partnerCx = samm.getMaterialGlobalAssetId();
```

Comment:
below variable is named `matNbrCatenaX` which might be more suitable. Please rename